SpatialGDK/Source/SpatialGDK/Public/SimulatedPlayers/SimulatedPlayer.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "SimulatedPlayer.generated.h"
+
+UCLASS()
+class SPATIALGDK_API USimulatedPlayer : public UBlueprintFunctionLibrary
+{
+	GENERATED_BODY()
+
+public:
+	/** Get whether this client is simulated.
```

Comment:
Nit - drop this one line down. The all text should be on a line starting with a single *